docs/docsite/rst/user_guide/windows_faq.rst

```diff
@@ -177,6 +177,8 @@ In addition, the following Ansible Core modules/action-plugins work with Windows
 * template (also: win_template)
 * wait_for_connection
 
+Note that as Ansible moves towards Collections more Windows modules have been added and they have migrated into several collections namespaces: `ansible.windows <https://docs.ansible.com/ansible/latest/collections/ansible/windows/index.html>`_, `community.windows <https://docs.ansible.com/ansible/latest/collections/community/windows/index.html>`_, and `chocolatey <https://docs.ansible.com/ansible/latest/collections/chocolatey/chocolatey/index.html#plugins-in-chocolatey-chocolatey>`_.
```

Comment:
```suggestion
Note that as Ansible moves towards Collections more Windows modules have been added and they have migrated into several collections namespaces: `ansible.windows <https://docs.ansible.com/ansible/latest/collections/ansible/windows/index.html>`_, `community.windows <https://docs.ansible.com/ansible/latest/collections/community/windows/index.html>`_, and `chocolatey <https://docs.ansible.com/ansible/latest/collections/chocolatey/chocolatey/index.html>`_.
```

Reply:
I'm going to resolve this particular suggested edits since the links should the :ref: instead. I'll post suggested edits for those next.